### ISSUE TITLE
Fast blocks without inducing red streaks

### ISSUE THREAD

**Issue by faddat** [osmosis-labs/osmosis#2815]
### Highly informal presentation on osmosis block times

**Problem:**

- we want fast blocks
- validators miss lots of blocks because we want fast blocks
- validators feel odd about missing lots of blocks because of the visualization of red streaks across the web

**Solution**

- don't change the commit timeout value
- use skip_timeout_commit=true — this will cause osmosis to make blocks as fast as possible, but vals will wait for other signers.  The maximum block time will remain the same as it is now. 
- If a validator is down, block times will get longer.  This will increase the pressure to do a great job of validating, because downtime will slow osmosis down.

**Context**:
https://twitter.com/blockpane/status/1572823202753089536

**Desired result**

This should allow osmosis to produce blocks very raipidly without causing osmosis validators to miss significantly more blocks. 

When a validator is down, osmosis will produce blocks at its current speed.  Theoretically speaking, this reactivity will increase the pressure on validators to stay up.

**Comment by ValarDragon** [comment]
I don;t think this makes sense. 

- 100% of votes coming in is a pretty silly metric
- Can lead to cases with very low sync advantage factors for full nodes, potentially even non-existent ones when accounting for hardware differences.

**Comment by gaia** [comment]
What's the rationale for using `skip_timeout_commit=false` as default? There seems to be no downside.

**Comment by faddat** [comment]
> 100% of votes coming in is a pretty silly metric

True; but I am not sure how to destigmatize missed blocks.  It took me 10 months to realize that it really didn't matter.  People shall meme this as long as the explorers show it and they probably should.

> Can lead to cases with very low sync advantage factors for full nodes, potentially even non-existent ones when accounting for hardware differences.

I think I still don't understand this, would you mind eli5ing?

thanks

**Comment by gaia** [comment]
> People shall meme this as long as the explorers show it and they probably should.

As opposed to %percentages, explorers could standardize on showing colors (green/yellow/red) and/or quality (excellent/good/fair/faulty) for uptime thresholds? Getting an influencer to have a talk about this on YT/Twitter would be a good starting point. But this is OT here.

**Comment by ValarDragon** [comment]
> What's the rationale for using skip_timeout_commit=false as default? There seems to be no downside.'

This is a misunderstanding of block time parameterization in decentralized networks,

You need full nodes on weaker hardware than validators to be able to sync to tip efficiently. This is impossible if you shave down this time.

You also lose any sort of predictability to block times.

**Comment by faddat** [comment]
So, I'm really not sure about the first part, but I fully agree on the second part about losing the ability to predict block time reliably.  Closing.
